agent/bench-scripts/pbench-specjbb2005

```diff
@@ -222,6 +222,7 @@ for i in `seq $start_warehouses $inc_warehouses $stop_warehouses`; do
 	pbench-postprocess-tools --group=$tool_group --iteration=$new_iteration_name --dir=$benchmark_results_dir/$new_iteration_name/reference-result
 	let iteration=$iteration+1
 done
+pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir
```

Comment:
Add "`end`" argument, and invoke it with the "`beg`" argument between lines 215 and 216.